src/views/Home/components/Mission/mission.scss

```diff
@@ -1,3 +1,108 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/responsiveness";
+@use "~stylesheets/typography";
+@use "~stylesheets/variables";
 
-/* Delete these lines when you begin development */
+.Mission {
+  position: relative;
+
+  /* Text Layer */
+  &--text {
+    display: flex;
+    flex-direction: column;
+    background: url("~images/mission-background.png");
+    background-size: 100%;
+    padding: 10% 22%;
```

Comment:
```    
    @include responsiveness.respond(phone) {
      padding: 10px 20px;
    }
```

Reply:
Personally, I think the existing iteration is better, because it doesn't overlap with the images:
![image](https://user-images.githubusercontent.com/33771623/101594517-ae815d00-39a6-11eb-8c41-6e5a835b4b82.png)

I would love to hear your feedback though - we can scale the `title-text` appropriately later.

Reply:
Generally, I would think it is ok to drop graphics on a limited size screen so more space can be used to display information, but this looks fine as well. Thanks for making the adjustments!

Reply:
No, yeah - that makes sense! @ilopezro & @mocklee, any opinions?

Reply:
apologies for not getting on this sooner.. I'm not opposed to leaving it as-is, the 3d effect works pretty well with how the font is rendering!

Reply:
I've added breakpoints to decrease spacing and slightly increase the container widths (even though it might be off-centered now). Let me know how it looks!

Reply:
This looks great, thanks for making the adjustments 🙂.